src/main/resources/controllers-context.xml

```diff
@@ -25,7 +25,7 @@
         <property name = "password" value = "${db.password}"/>
     </bean>
 
-    <context:property-placeholder location="file:C:\Users\Iuliia_Gorshkova\Desktop\Dog\src\main\resources\application.properties"/>
+    <context:property-placeholder location="file:application.properties"/>
```

Comment:
Um.. Did you check it? "file:" means it's going to look for the file on file system in your working directory. You need to look for it in the classpath.